### PR TITLE
[node-manager] Add instruction how add static node to cluster

### PR DESCRIPTION
**Pull request by d2285**
## Description

add description how add static node to cluster.

## Why do we need it, and what problem does it solve?

this describe how add static node to cluster when UsePAM must set in "yes" and generate pair of SSH keys with an empty passphrase on server.

## What is the expected result?

static node bootstrapped into the cluster when UsePAM set in "yes" on node and generate pair of SSH keys with an empty passphrase  on server.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: node-manager
type: fix 
summary: Add instruction how add static node to cluster
impact_level: default
```

### CODE REVIEW

**Review by mortelumina98 — APPROVED** [review]

**Review by Zhbert — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: modules/040-node-manager/docs/FAQ_RU.md [hunk 1]
```diff
@@ -1128,18 +1196,17 @@ metadata:
 
 ## Что такое ресурс Instance?
 
-Ресурс Instance содержит описание объекта эфемерной машины, без учета ее конкретной реализации. Например, машины созданные MachineControllerManager или Cluster API Provider Static будут иметь соответствующий ресурс Instance.
+Ресурс `Instance` в Kubernetes представляет собой описание объекта временной виртуальной машины, но без конкретной реализации. Это абстракция, которая используется для управления машинами, созданными с помощью таких инструментов, как MachineControllerManager или Cluster API Provider Static.
```
Comment (mortelumina98) [inline modules/040-node-manager/docs/FAQ_RU.md:1199]:
```suggestion
Ресурс `Instance` в Kubernetes представляет собой описание объекта эфемерной виртуальной машины, но без конкретной реализации. Это абстракция, которая используется для управления машинами, созданными с помощью таких инструментов, как MachineControllerManager или Cluster API Provider Static.
```

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [10811](https://github.com/deckhouse/deckhouse/pull/10811) to the branch [release-1.66](https://github.com/deckhouse/deckhouse/tree/release-1.66) successful!

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [10812](https://github.com/deckhouse/deckhouse/pull/10812) to the branch [release-1.65](https://github.com/deckhouse/deckhouse/tree/release-1.65) successful!

**Pull request merged**